cdc/cdc.cc

```diff
@@ -298,7 +298,60 @@ class transformer final {
         auto shard_id = dht::murmur3_partitioner::shard_of(
                 t, _ctx._snitch->get_shard_count(*ep), _ignore_msb_bits);
         mutation res(_log_schema, stream_id(ep->addr(), shard_id));
-        set_pk_columns(m.key(), 0, res);
+        auto& p = m.partition();
+        if(p.partition_tombstone()) {
+            // Partition deletion
+            set_pk_columns(m.key(), 0, res);
+        } else if (!p.row_tombstones().empty()) {
+            // range deletion
+            int batch_no = 0;
+            for (auto& rt : p.row_tombstones()) {
+                auto set_bound = [&] (const clustering_key& log_ck, const clustering_key_prefix& ckp) {
```

Comment:
Shouldn't we have an indication here of the operation (range delete start/end)? Also whether the bound is inclusive or exclusive.

Reply:
The operation type is added by Calle in the following patchset. There start and end are easily distinguished.
For inclusive/exclusive, that's something we have to do. The question is how. There are some options. One is to make the operation type RT_start_inclusive, RT_start_exclusive, RT_end_inclusive and RT_end_exclusive. It's probably the simplest. Another would be an addition of an extra column to mark inclusiveness but I like the first option more. What do you think @avikivity? Also FYI @elcallio

Reply:
inclusive/exclusive markers is symmetric with clustering bounds so makes most sense. Ii.e. agree on first option)